pkg/mobile/mobileClient_test.go

```diff
@@ -0,0 +1,62 @@
+package mobile
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+func TestMobileHandler(t *testing.T) {
+	t.Run("test mobile handler", func(t *testing.T) {
+		handler = &MobileHandler{}
+		watcher1 := MobileWatcher{events: make(chan watch.Event)}
+		watcher2 := MobileWatcher{events: make(chan watch.Event)}
+		handler.watchers = append(handler.watchers, watcher1)
+		handler.watchers = append(handler.watchers, watcher2)
```

Comment:
It's possible to forget assign the array back in this way. So given that there is a "RemoveWatcher" method now, maybe worth moving this into a `AddWatcher` method?